vllm/engine/arg_utils.py

```diff
@@ -373,12 +373,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             action='store_true',
                             help='Disables sliding window, '
                             'capping to sliding window size')
-        parser.add_argument(
-            '--use-v2-block-manager',
-            default=EngineArgs.use_v2_block_manager,
-            action='store_true',
-            help='Use BlockSpaceMangerV2. By default this is set to True. '
-            'Set to False to use BlockSpaceManagerV1')
+        parser.add_argument('--use-v2-block-manager',
+                            action='store_true',
+                            help='[DEPRECATED] block manager v1 has been '
+                            'removed and block manager v2 is now the default '
```

Comment:
nit - Do you want to call out BlockSpaceManagerV2 is same as  SelfAttnBlockSpaceManager by saying something like "and block manager v2 (SelfAttnBlockSpaceManager) is now the default " since we have renamed the class.

Reply:
Good point --- just adjusted the docstring.